### DISCUSSION TITLE
Custom Header Sticky

### DISCUSSION THREAD

**Discussion by bjoe87** [petyosi/react-virtuoso#513] [Q&A] [answered]
Hi!

Was curious if its possible to have a sticky header appear above the list items in terms of html markup? I do realize there is a way to customize the header to have it appear above the markup but the docs imply this was to always be a non-sticky header. Main use case for this is  keyboard accessibility. 

Thanks for your time!

**Comment by petyosi** [comment, marked as answer]
You can use a topItems={1} and customize the first item. Not the most elegant workaround, but it should work.

**Reply by bjoe87** [reply]
Thanks for such a fast response! Awesome to see. :) Unfortunately the markup has the sticky item being below the list of items. So if there are any items in the list. The user will have to tab through the list first before hitting the sticky item. 
![Markup](https://user-images.githubusercontent.com/25537481/144647277-c632f8fa-ba18-41a7-98e0-c35bedeb3ec0.png)

Again thanks for the fast response and awesome component!

**Reply by petyosi** [reply]
why don't you work around that with a `tabindex`?

**Reply by bjoe87** [reply]
Yeah probably can work with that. Just want to try to avoid anything that would give accessibility tooling problems. Thanks again. I thought maybe I was missing something but this will help me determine what I need to do.
